packages/core/src/exchanges/debug.ts

```diff
@@ -2,7 +2,7 @@ import { pipe, tap } from 'wonka';
 import { Exchange } from '../types';
 
 export const debugExchange: Exchange = ({ forward }) => {
-  if (process.env.NODE_ENV === 'production') {
+  if (process.env && process.env.NODE_ENV === 'production') {
```

Comment:
Is this an accidental change? It likely won’t fix any issue 😅

Reply:
Oops, yes I was experimenting with a plugin to produce full-browser builds 😅